tests/testthat/helper.R

```diff
@@ -133,8 +133,10 @@ test_optimizer = function(optimizer, param_set = list(), n_dim, term_evals = 2L,
   optimizer$optimize(instance)
   archive = instance$archive
 
-  expect_data_table(archive$data(), nrows = real_evals)
-  expect_equal(instance$archive$n_evals, real_evals)
+  if(!is.na(real_evals)) {
```

Comment:
This line needs a comment

Reply:
Removed. I deactivated the internal termination for the unit tests now.